pages/main-page.ts

```diff
@@ -11,6 +11,8 @@ export class MainPage {
         this.accessStorageLink = page.getByRole('link', { name: 'Storage' });
         this.accessUsersLink = page.getByRole('link', { name: 'Users' });
         this.installButton = page.getByRole("button", { name: "Install", exact: true });
+        this.installationSize = page.getByText("Installation will take");
+        this.noUserAdd = page.getByText('No user defined yet');
```

Comment:
It missing the instance fields declaration as `readonly`:
```js
readonly installationSize: Locator;
readonly noUserAdd: Locator;
```
See [readonly](https://www.typescriptlang.org/docs/handbook/2/classes.html#readonly)